Neos.Kickstarter/Resources/Private/Generator/Translations/TranslationsTemplate.xlf.tmpl

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+	<file original="" product-name="{packageKey}" source-language="{languageKey}" datatype="plaintext">
+		<body>
+			<trans-unit id="some.id" xml:space="preserve">
+				<source>Some label</source>
```

Comment:
I think the idea is to just generate the base file for the default translation at this time, so it's totally fine to skip `target`.